docs/WhyIWYUIsDifficult.md

```diff
@@ -136,28 +129,25 @@ Another case is a header file like this:
 
 We might think we can remove an `#include` of `foo.h` and replace it by `#include "module_writer.h"`, but that is likely to break the build if `module_writer.h` requires `MODULE_NAME` be defined.  Since my file doesn't participate in this dependency at all, it won't even notice it.  IWYU needs to keep track of dependencies between files it's not even trying to analyze!
 
-
-### Private Includes ###
+## Private Includes ##
 
 Suppose you write `vector<int> v;`. You are using vector, and thus have to `#include <vector>`. Even this seemingly easy case is difficult, because vector isn't actually defined in `<vector>`; it's defined in `<bits/stl_vector.h>`. The C++ standard library has hundreds of private files that users are not supposed to `#include` directly. Third party libraries have hundreds more.  There's no general way to distinguish private from public headers; we have to manually construct the proper mapping.
 
 In the future, we hope to provide a way for users to annotate if a file is public or private, either a comment or a `#pragma`. For now, we hard-code it in the IWYU tool.
 
 The mappings themselves can be ambiguous. For instance, `NULL` is provided by many files, including `stddef.h`, `stdlib.h`, and more. If you use `NULL`, what header file should IWYU suggest? We have rules to try to minimize the number of `#includes` you have to add; it can get rather involved.
 
-
-### Unparsed Code ###
+## Unparsed Code ##
 
 Conditional `#includes` are a problem for IWYU when the condition is false:
 
     #if _MSC_VER
     #include <foo>
     #endif
 
-If we're not running under windows (and IWYU does not currently run under windows), we have no way of telling if foo is a necessary `#include` or not.
-
+Since we're not running using the Microsoft Visual C++ compiler (IWYU is built on top of Clang), we have no way of telling if foo is a necessary `#include` or not.
```

Comment:
I'd like for this to be a separate commit from all the formatting changes, as it makes content changes

Reply:
Sure.